src/openms/source/METADATA/ID/IdentificationDataConverter.cpp

```diff
@@ -783,7 +783,7 @@ namespace OpenMS
     const map<IdentificationData::ScoreTypeRef, Size>& scores,
     map<Size, MzTabParameter>& output)
   {
-    for (const pair<IdentificationData::ScoreTypeRef, Size>& score_pair :
+    for (const pair<const IdentificationData::ScoreTypeRef, Size>& score_pair :
```

Comment:
I would be finde with const auto&

Reply:
Yes true. I did not address "autos" yet since it is hard to decide automatically when it helps readability and when it doesnt.